### PR TITLE
Fixes #1098 support for inputs

### LINKED ISSUE #1098: PKSIm.CLI: Support for input in qualifications

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Assets/PKSimConstants.cs [hunk 1]
```diff
@@ -885,6 +885,8 @@ public static string MapToSnapshotNotSupportedWithoutContext(string snapshotType
 
          public static string CannotLoadSnapshotFromFile(string fileFullPath) => $"TODO Cannot load snapshot from file '{fileFullPath}'. Please make sure that the file exists and that it is a valid snapshot file.";
 
+         public static string CannotFindBuildingBlockInSpahsot(string buildingBlockType, string buildingBlockName, string snapshotName) => $"Could not find {buildingBlockType} '{buildingBlockName}' in snapshot '{snapshotName}'.";
```
Comment (Yuri05) [inline src/PKSim.Assets/PKSimConstants.cs]: Spahsot? ;)

File: src/PKSim.Core/CoreConstants.cs [hunk 2]
```diff
@@ -46,6 +46,7 @@ public static class Filter
          public const string USER_TEMPLATE_DB_EXTENSION = ".templateDbUser";
          public const string SYSTEM_TEMPLATE_DB_EXTENSION = ".templateDbSystem";
          public const string GENE_DB_EXTENSION = ".expressionDb";
+         public static readonly string MARDOWN_EXTENSION = ".md";
```
Comment (Yuri05) [inline src/PKSim.Core/CoreConstants.cs]: Mar**k**down

Comment (msevestre) [reply src/PKSim.Core/CoreConstants.cs]: going to fix shortly

File: src/PKSim.CLI.Core/Services/QualificationRunner.cs [hunk 3]
```diff
@@ -133,36 +152,64 @@ private IEnumerable<PlotMapping> retrieveChartDefinitionsForSimulation(Simulatio
...
+         var buildingBlock = project.BuildingBlockByName(input.Name, input.Type);
+
+         var inputsFolder = configuration.InputsFolder;
+         var projectName = FileHelper.RemoveIllegalCharactersFrom(project.Name);
+         var buildingBlockName = FileHelper.RemoveIllegalCharactersFrom(input.Name);
+         var targetFolder = Path.Combine(inputsFolder, projectName, input.Type.ToString());
+         DirectoryHelper.CreateDirectory(targetFolder);
+
+         var fileFullPath = Path.Combine(targetFolder, $"{buildingBlockName}{CoreConstants.Filter.MARDOWN_EXTENSION}");
+         await _markdownReporterTask.ExportToMarkdown(buildingBlock, fileFullPath);
```
Comment (msevestre) [inline src/PKSim.CLI.Core/Services/QualificationRunner.cs]: This is a dummy implementation of the markdwon reporter for now until we know more about what we need!

File: src/PKSim.Core/CoreConstants.cs (hunk 2 above)
Comment (msevestre) [reply src/PKSim.Core/CoreConstants.cs]: done. This should probably be move to Core anyways. We'll make a note of that and do it when movign sutff to core

**Pull request merged**